treeherder/intermittents_commenter/commenter.py

```diff
@@ -452,15 +450,12 @@ def build_bug_map(self, bugs, option_collection_map, start_day, end_day):
             if test_name:
                 manifest = all_tests[test_name][0]
                 if manifest:
-                    """
-                    # Bug 1972307 - disabled due to no comments
-                    tasks_count = self.get_task_labels_and_count(manifest, start_day, end_day)
-                    job_name = bug["job__signature__job_type_name"]
+                    tasks_count = self.get_task_labels_and_count(manifest, days)
```

Comment:
I think this could be optimized to pass in job_name (without -<X>) so we only pass back count (or zero if not found).